cmd/booster-http/server.go

```diff
@@ -151,122 +124,59 @@ func (s *HttpServer) handleIndex(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(idxPage)) //nolint:errcheck
 }
 
-func (s *HttpServer) handlePieceRequest(w http.ResponseWriter, r *http.Request) {
+func (s *HttpServer) handleInfo(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	v := apiVersion{
+		Version: "0.2.0",
+	}
+	json.NewEncoder(w).Encode(v) //nolint:errcheck
+}
+
+func (s *HttpServer) handleByPieceCid(w http.ResponseWriter, r *http.Request) {
+	startTime := time.Now()
+	ctx, span := tracing.Tracer.Start(r.Context(), "http.piece_cid")
+	defer span.End()
+	stats.Record(ctx, metrics.HttpPieceByCidRequestCount.M(1))
+
 	q, err := url.ParseQuery(r.URL.RawQuery)
 	if err != nil {
 		msg := fmt.Sprintf("parsing query: %s", err.Error())
 		writeError(w, r, http.StatusBadRequest, msg)
+		stats.Record(ctx, metrics.HttpPayloadByCid400ResponseCount.M(1))
```

Comment:
I'm not familiar with all of the metrics vars, but as we're only doing piece retrieval by piece cid shouldnt this be `HttpPieceByCid400ResponseCount` or similar?

Same for the below values.